### PR TITLE
Html-css-popup

### PR DESCRIPTION
**Pull request by Nazar-cmd**
### Html-css-popup

[Demo](https://nazar-cmd.github.io/popup/) |
[Code base](https://github.com/Nazar-cmd/popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+@media (prefers-reduced-motion: reduce) {
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    transition-duration: 0.01ms !important;
+    scroll-behavior: auto !important;
+  }
+}
```
Comment (madmaxWMFU) [inline submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss:70]: Add an empty line

File: submissions/Nazar-cmd/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+
+
+<main>
+    <a href="#">
+
+    </a>
+</main>
+</body>
+</html>
```
Comment (madmaxWMFU) [inline submissions/Nazar-cmd/html-css-popup/index.html:130]: Add an empty line.

File: submissions/Nazar-cmd/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    <meta name="keywords" content="HTML, SCSS, Popup">
+    <meta name="author" content="Nazar">
+    <title>html-css-popup</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="icon" href="./assets/icons/favicon.png" type="image/png">
+</head>
+<body>
+<header>
+    <nav class="navbar" >
+        <a href="#" class="navbar-link">
```
Comment (madmaxWMFU) [inline submissions/Nazar-cmd/html-css-popup/index.html:16]: Navigation is best used in this format.
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work!
See comments below

**Comment by madmaxWMFU** [comment]
Сorrect mistakes from [HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fnazar-cmd.github.io%2Fpopup%2F) and [CSS validator](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fnazar-cmd.github.io%2Fpopup%2F&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=ru)

File: submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+/* A elements that don't have a class get default styles */
+a:not([class]) {
+  text-decoration-skip-ink: auto;
+}
+
+/* Make images easier to work with */
+img,
+picture {
```
Comment (madmaxWMFU) [inline submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss:47]: Avoid naming by id or tag, better to use the class name.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

File: submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss (hunk 4 above)
Comment (Nazar-cmd) [reply submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss:47]: But this is reset styles, I want to cover all page with this. Also what is a purpose of a new line in the end of each document? Thank you)

**Comment by Nazar-cmd** [comment]
All validators have been passed and wishes fulfilled)

File: submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss (hunk 4 above)
Comment (madmaxWMFU) [reply submissions/Nazar-cmd/html-css-popup/css/reset_styles.scss:42]: Didn't notice

**Review by madmaxWMFU — APPROVED** [review]

**Comment by Nazar-cmd** [comment]
Is it something, that should be fixed?

**Comment by madmaxWMFU** [comment]
@MaxKalashnyk can you review it?

**Review by MaxKalashnyk — APPROVED** [review]
LGTM!

**Pull request merged**